src/main/java/pack/model/Address.java

```diff
@@ -0,0 +1,119 @@
+package pack.model;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.persistence.SequenceGenerator;
+
+import org.springframework.stereotype.Component;
+
+@Component
+@Entity
+public class Address {
+	@Id
+	@GeneratedValue(strategy=GenerationType.AUTO, generator="address_seq")
+	@SequenceGenerator(name="address_seq", sequenceName="address_seq", initialValue = 1, allocationSize = 100)
+	private int addressId;	
+	@Column
+	private String street;
+	@Column
+	private String suite;
+	@Column
+	private String city;
+	@Column
+	private String zipcode;
+	
+	
+	@OneToOne(cascade=CascadeType.ALL)
+	//@JoinColumn(name = "personId", nullable = false)
+	private Person personAddress;
+	
+	@OneToOne(mappedBy = "geoAddress",cascade=CascadeType.ALL)
+	private Geo geo;
+
+	public Address(){}
+	
+
+	public Address(String street, String suite, String city, String zipcode,Person personAddress,Geo geo) {
+		this.street = street;
+		this.suite = suite;
+		this.city = city;
+		this.zipcode = zipcode;
+		this.geo=geo;
+		this.personAddress=personAddress;
+		
+	}
+
+
```

Comment:
Please format your code, remove extra lines. In eclipse go to Preferences, Java, Editor, Save Actions and check Perform the selected actions on save, and check format souce code, format all lines.